photon-client/src/main/scala/com/linkedin/photon/ml/hyperparameter/tuner/AtlasTuner.scala

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
Bit unclear on this part. So the AtlasTuner will move out of photon-ml correct? and photon-ml will only contain the DummyTuner?

Reply:
I think both `DummyTuner` and `AtlasTuner` will stay in photon-ml and li-photon-ml because we want to keep the code exactly the same. And we will use `HyperparameterTunerName` to choose which one to be called.

Reply:
Will change it. Thank you!